src/test/munit/sftp-write-test-case.xml

```diff
@@ -68,4 +68,52 @@
         </munit:validation>
     </munit:test>
 
+    <munit:test name="sftp-write-on-invalid-working-directory" expectedErrorType="SFTP:ILLEGAL_PATH">
+        <munit:behavior>
+            <set-variable variableName="fileName" value="filePath.txt"/>
+        </munit:behavior>
+        <munit:execution>
+            <sftp:write config-ref="config-invalid-folder" path='#[vars.fileName]' createParentDirectories="false">
+                <sftp:content>
+                    #[%dw 2.0
+                    output application/java
+                    ---
+                    "asd"]
+                </sftp:content>
+            </sftp:write>
+        </munit:execution>
+    </munit:test>
+
+    <munit:test name="sftp-write-on-invalid-path-directory" expectedErrorType="SFTP:ILLEGAL_PATH">
+        <munit:behavior>
+            <set-variable variableName="fileName" value="invalid/filePath.txt"/>
+        </munit:behavior>
+        <munit:execution>
+            <sftp:write config-ref="config" path='#[vars.fileName]' createParentDirectories="false">
+                <sftp:content>
+                    #[%dw 2.0
+                    output application/java
+                    ---
+                    "asd"]
+                </sftp:content>
+            </sftp:write>
+        </munit:execution>
+    </munit:test>
+
+    <munit:test name="sfto-write-on-invalid-absolute-path-equals-to-working-directory" expectedErrorType="SFTP:ILLEGAL_PATH">
```

Comment:
I am not too sure about the name of this test, what do you mean by "equals-to-working-directory" at the end? I would leave it at "sfto-write-on-invalid-absolute-path". Plus, typo in "sftO" :P

Reply:
Fixed